azure-pipelines.yml

```diff
@@ -238,6 +248,8 @@ jobs:
           testGroup: outerloop-jitstress-isas-arm
         ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-jitstress-isas-x86') }}:
           testGroup: outerloop-jitstress-isas-x86
+        ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-jitstress-regs-x86') }}:
```

Comment:
I added a separator in the names based on the feedback from @sandreenko in the last PR: https://github.com/dotnet/coreclr/pull/24630#pullrequestreview-239050387

This does make it not match the outer `jitstressregs` jobs, however. It would likely be worth making them all consistent, and I don't have a preference either way, but the existing build-definitions can't simply be renamed due to other branches/PRs actively using them.